WhiteDentalClinic/WhiteDentalClinic.Api/Controllers/AppointmentController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+using WhiteDentalClinic.Application.Models;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Services.Interfaces;
+
+namespace WhiteDentalClinic.Api.Controllers
+{
+    public class AppointmentController : ApiController
+    {
+        readonly IAppointmentService _appointmentService;
+        public AppointmentController(IAppointmentService appointmentService)
+        {
+            _appointmentService = appointmentService;
+        }
+
+        [HttpGet("allapp/customer{customerRequestId}")]
```

Comment:
I think something like "appointments/customers/{customerId}" as an endpoint path would be much better as this is usually the standard used and it's easier to understand.